tests/unittests/GraphOptzTest.cpp

```diff
@@ -1817,6 +1825,86 @@ TEST_F(GraphOptz, foldQuantizeIntoVarMultipleUsages) {
   }
 }
 
+static SaveNode *getUniqueSaveNode(Function *F) {
```

Comment:
nit: doxygen. Prefer to note here this call does some checking itself/may fail the test.

Reply:
Done.